content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -285,9 +285,9 @@ If you restrict `list` or `watch` by resourceName, clients must include a `metad
 For example, `kubectl get configmaps --field-selector=metadata.name=my-configmap`
 {{< /note >}}
 
-Rather than referring to individual `resources` and `verbs` you can use the wildcard `*` symbol to refer to all such objects.
-For `nonResourceURLs` you can use the wildcard `*` symbol as a suffix glob match and for `apiGroups` and `resourceNames` an empty set means that everything is allowed.
-Here is an example that allows access to perform any current and future action on all current and future resources (note, this is similar to the built-in `cluster-admin` role).
+Rather than referring to individual `resources` 、`apiGroups`and `verbs` you can use the wildcard `*` symbol to refer to all such objects. 
+For `nonResourceURLs`, you can use the wildcard `*` symbol as a suffix glob match. For `resourceNames`, an empty set means that everything is allowed. In addition, if 'apiGroups' is an empty set, it can only access the resources of the core group. 
```

Comment:
> In addition, if 'apiGroups' is an empty set, it can only access the resources of the core group.

This is not accurate. The `apiGroups` list must contain `""` (for example, `[""]`) to access the core group. The empty set (`[]`) is invalid and prevented by validation.

Reply:
All changed